bin/run_lint

```diff
@@ -259,23 +271,60 @@ fi
 top_dir=$(cd $(dirname $(dirname $0)) && pwd)
 rv=0
 
+# Print version information
+cppcheck --version
+echo "$(pylint3 --version | head -1)"
+echo "pycodestyle $(pycodestyle --version)"
+
 # Run pycodestyle or pylint or both for each module.
-# Modules should be ranked in decreasing likelihood of having lint errors.
 
 # Language:                                Python  | C
 # Checks:                           pycodestyle    | pycodestyle
 #                                           pylint |    cppcheck
-export PYTHONPATH=$PYTHONPATH:$top_dir/examples/common/python
-lint_module tests                    "$SINCE" 1  0   0  0 || rv=1
-lint_module enclave_manager          "$SINCE" 1  0   0  0 || rv=1
-lint_module examples/common          "$SINCE" 1  0   0  0 || rv=1
-lint_module examples/apps            "$SINCE" 1  0   0  1 || rv=1
-lint_module tc                       "$SINCE" 1  0   0  1 || rv=1
-lint_module common                   "$SINCE" 1  0   0  1 || rv=1
-lint_module docs/workload-tutorial   "$SINCE" 0  0   0  1 || rv=1
-lint_module sdk                      "$SINCE" 1  0   0  0 || rv=1
-lint_module listener                 "$SINCE" 1  0   0  0 || rv=1
-lint_module shared_kv_storage        "$SINCE" 1  0   0  1 || rv=1
-lint_module blockchain_connector     "$SINCE" 1  0   0  0 || rv=1
-
-exit $rv
+
+for MODULE in $MODULES
+do
+    case $MODULE in
+        tests)
+                lint_module tests                    "$SINCE" 1  0   0  0 || rv=1
+                ;;
+        enclave_manager)
+                lint_module enclave_manager          "$SINCE" 1  0   0  0 || rv=1
+                ;;
+        tc)
+                lint_module tc                       "$SINCE" 1  0   0  1 || rv=1
+                ;;
+        common)
+                lint_module common                   "$SINCE" 1  0   0  1 || rv=1
+                ;;
+        docs_workload_tutorial)
+                lint_module docs/workload-tutorial   "$SINCE" 0  0   0  1 || rv=1
+                ;;
+        sdk)
+                lint_module sdk                      "$SINCE" 1  0   0  0 || rv=1
+                ;;
+        listener)
+                lint_module listener                 "$SINCE" 1  0   0  0 || rv=1
+                ;;
+        shared_kv_storage)
+                lint_module shared_kv_storage        "$SINCE" 1  0   0  1 || rv=1
+                ;;
+        blockchain_connector)
+                lint_module blockchain_connector     "$SINCE" 1  0   0  0 || rv=1
+                ;;
+        examples)
+                lint_module examples                 "$SINCE" 1  0   0  1 || rv=1
```

Comment:
Is entire code under examples directory lint compliant ? There are lot of applications including 3rd party applications in examples directory

Reply:
The lint script would do a nested search for all .py and .cpp files and run lint on them.